executor/agents/agno/agno_agent.py

```
@@ -35,6 +35,28 @@
+        if hasattr(obj, "__dict__"):
+            return str(obj)
+        # Handle objects with model_dump method (Pydantic models)
+        if hasattr(obj, "model_dump"):
+            return str(obj)
+        # Handle objects with to_dict method
+        if hasattr(obj, "to_dict"):
+            return str(obj)
+        # Fallback: convert to string representation
+        try:
+            return str(obj)
+        except Exception:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)